packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -264,13 +281,20 @@ export function dispatchEvent(
   );
 }
 
-// Attempt dispatching an event. Returns a SuspenseInstance or Container if it's blocked.
-export function attemptToDispatchEvent(
+export let return_shouldDispatch = false;
+export let return_targetInst = null;
```

Comment:
Technically this leaks at module level but next event will clear it. So it's fine.